src/coreComponents/physicsSolvers/wavePropagation/AcousticWaveEquationSEM.cpp

```diff
@@ -339,19 +343,21 @@ void AcousticWaveEquationSEM::computeSeismoTrace( real64 const time_n, real64 co
           // TODO: remove saveSeismo and replace with TimeHistory
           for( localIndex iSample = 0; iSample < m_nsamplesSeismoTrace; ++iSample )
           {
-            this->saveSeismo( iSample, p_rcvs[iSample][ircv], GEOSX_FMT( "seismoTraceReceiver{:03}.txt", ircv ) );
+            this->saveSeismo( iSample, var_rcvs[iSample][ircv], GEOSX_FMT( "seismoTraceReceiver{:03}.txt", ircv ) );
           }
         }
       }
     } );
   }
+
 }
 
 /// Use for now until we get the same functionality in TimeHistory
-void AcousticWaveEquationSEM::saveSeismo( localIndex iSeismo, real64 valPressure, string const & filename )
+/// TODO: move implementation into WaveSolverBase
+void AcousticWaveEquationSEM::saveSeismo( localIndex const iSeismo, real64 val, string const & filename )
```

Comment:
val is also const, no ?

Reply:
The values are copied anyway for simple types. 
I would omit them for all simple types, but that may not be the policy in Geosx.

Reply:
Indeed, it's juste to be coherent between iSeismo and val.

Reply:
I agree, I'll add _const_ for clarity.